geometry/include/pcl/geometry/mesh_indices.h

```diff
@@ -159,422 +160,55 @@ class VertexIndex
   /** \brief Stored index. */
   int index_;
 
-  friend std::istream&
-  operator>>(std::istream& is, pcl::geometry::VertexIndex& index);
+  friend std::istream& operator>>
+      <IndexTagT>(std::istream& is, MeshIndex<IndexTagT>& index);
```

Comment:
From the linked answer, do you think this is a viable alternate?
```suggestion
      <>(std::istream& is, MeshIndex<IndexTagT>& index);
```

Reply:
If I remember correctly yes, this works! I can try it and change it on the fly if you want.